webapp/app/routes.py

```diff
@@ -186,11 +186,14 @@ def add_http_header(response):
                                                  'payment=(), picture-in-picture=(), publickey-credentials-get=(), ' \
                                                  'screen-wake-lock=(), sync-xhr=(), usb=(), web-share=(), ' \
                                                  'xr-spatial-tracking=() '
+        additional_connect_src = ""
+        if request.path == '/unlock_code_request/step/unlock_code_success':
```

Comment:
Why is this only for this one route?

Reply:
Because this is the only page (where the NewsletterBox component is embedded) where we do this JS fetching from the react component to a backend route